### ISSUE TITLE
View FSMs from other game?

### ISSUE THREAD

**Issue by war40k** [nesrak1/FSMViewAvalonia#15]
I was trying to view FSMs from another game but didn't know where to get started anytime I set the directory to any other loc the app would crash. No trouble viewing HK though

**Comment by nesrak1** [comment]
What version is the engine? May be able to fix this with a classdata.tpk swap.

**Comment by war40k** [comment]
It was 2020.3.2 is it something simple I could do?

**Comment by nesrak1** [comment]
That's right on the cutoff on the classdata.tpk in this repo. Give me some time to look into it.

**Comment by nesrak1** [comment]
Sorry about the wait on this. Could you get me a stack trace on this? Think this should work: https://github.com/nesrak1/UABEA/wiki/Sending-a-crashlog
